src/components/Services.astro

```diff
@@ -0,0 +1,71 @@
+<section>
+  <div class="services-information">
+    <h2>Usługi</h2>
+    <p class="large-body">
+      Prowadzimy kompleksowy serwis, wykonywany z&nbspnajwyższą starannością.
+      Mamy szeroki wachlarz usług z&nbspktórych możemy wyróżnić:
+    </p>
+  </div>
+
+  <ul class="grid-3-col">
+    <li class="service-1">
+      <img src="img/service_1.jpg" alt="" />
+      <div class="small-body-decoration">
+        <p class="small-body">
+          Naprawy siłowników nierozbieralnych, zakuwanych, aluminiowych
+          i&nbspstalowych.
+        </p>
+      </div>
+    </li>
```

Comment:
repeatable content, try to implement a small component with the card (?) and iterate over array here

Reply:
https://docs.astro.build/en/core-concepts/astro-components/

components documentation

Reply:
https://docs.astro.build/en/core-concepts/astro-components/#component-props

Compoennt props (preferably with types since we are using TS)